images/nri-kube-events/configs/latest.apko.yaml

```diff
@@ -0,0 +1,34 @@
+contents:
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - ca-certificates-bundle
+    - busybox
+    - newrelic-nri-kube-events
+    - wolfi-baselayout
+    - bind-tools
+    - tini
+
+accounts:
+  groups:
+    - groupname: nri-kube-events
+      gid: 65532
+  users:
+    - username: nri-kube-events
+      uid: 65532
+      gid: 65532
+  run-as: 65532
+
+entrypoint:
+  command: /sbin/tini -- /usr/bin/nri-kube-events
+
+archs:
```

Comment:
Do we need this environment variable? 
<img width="498" alt="image" src="https://github.com/chainguard-images/images/assets/1714486/22809e12-7621-4080-80e6-7ae2354354b5">

https://github.com/newrelic/nri-kube-events/blob/main/Dockerfile

Reply:
it should be okay right now.